nipype/pipeline/plugins/multiproc.py

```diff
@@ -49,41 +75,93 @@ class MultiProcPlugin(DistributedPluginBase):
 
     """
 
+    def _init_worker(self):
+        """
+        Ignore interrupt signal, handle in main thread
+        https://noswap.com/blog/python-multiprocessing-keyboardinterrupt
+        """
+        signal.signal(signal.SIGINT, signal.SIG_IGN)
```

Comment:
The interruption signal handling is a WIP:
https://noswap.com/blog/python-multiprocessing-keyboardinterrupt
http://jtushman.github.io/blog/2014/01/14/python-%7C-multiprocessing-and-interrupts/
http://jessenoller.com/blog/2009/01/08/multiprocessingpool-and-keyboardinterrupt